lib/Server.php

```diff
@@ -323,7 +335,7 @@ private function onAcceptable(string $watcherId, $server) {
             return;
         }
 
-        \assert($this->logDebug("accept {$peerName}"));
+        \assert($this->logDebug("accept {$peerName} #" . (int) $client));
```

Comment:
Should probably be `"Accepting TCP connection from {$peerName} #" . (int)  $client`